litespi/spi_nor_features.py

```diff
@@ -0,0 +1,75 @@
+import enum
+
+
+class SpiNorFeatures(enum.Flag):
+    FEATURE_ERASED_ZERO     = enum.auto()
+    """Erasing flash ends up with 0x00 rather than 0xff."""
+
+    FEATURE_NO_ERASE        = enum.auto()
+    """Chip does not support erase commands."""
+
+    FEATURE_OTP             = enum.auto()
+    """Has a one time programming area."""
+
+    FEATURE_WRSR_EWSR       = enum.auto()
+    """Before sending the WRSR instruction the Enable-Write-Status-Register must be written too."""
+
+    FEATURE_WRSR_WREN       = enum.auto()
+    """Before sending WRSR instruction, the Write Enable (WREN) instruction must be decoded and executed to set the Write Enable Latch (WEL) bit in advance."""
+
+    FEATURE_WRSR_EITHER     = FEATURE_WRSR_EWSR | FEATURE_WRSR_WREN
+
+    FEATURE_SINGLE          = enum.auto()
```

Comment:
Can we just have a "supported opcodes" type thing or "supported read opcodes"?

Reply:
Will be done in the next PR